packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -7,9 +7,14 @@ import { handleParamGovernance, ParamTypes } from '@agoric/governance';
 import { offerTo } from '@agoric/zoe/src/contractSupport/index.js';
 
 import { AMM_INSTANCE } from './params.js';
+import { makeTracer } from '../makeTracer.js';
+
+const trace = makeTracer('Reserve', true);
 
 const makeLiquidityKeyword = keyword => `${keyword}Liquidity`;
 
+const RunKW = 'RUN';
```

Comment:
Interesting... I have put my toe in the water with constants for keywords... I'm not entirely happy with the results. let's see how it goes here...

Reply:
Did you consider something like `StableKW` where only the value would change when we go to IST?

Reply:
I like that idea and it didn't occur to me. I wouldn't want to do it in just this PR but I'll think about it for https://github.com/Agoric/agoric-sdk/issues/4800

IST seems so central to the package that my intuition is abstracting it would just add cognitive load and provide no benefit. I see it could be beneficial in making the transition easier, but that's a one-shot change that shouldn't burden regular development.